marker/schema/blocks/basetable.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from marker.schema import BlockTypes
+from marker.schema.blocks import Block, BlockOutput
+from marker.schema.blocks.tablecell import TableCell
+
+
+class BaseTable(Block):
+    block_type: BlockTypes | None = None
+    html: str | None = None
+
+    def format_cells(self, document, child_blocks):
+        child_cells: List[TableCell] = [document.get_block(c.id) for c in child_blocks]
+        unique_rows = sorted(list(set([c.row_id for c in child_cells])))
+        html_repr = "<table><tbody>"
+        for row_id in unique_rows:
+            row_cells = sorted([c for c in child_cells if c.row_id == row_id], key=lambda x: x.col_id)
+            html_repr += "<tr>"
+            for cell in row_cells:
+                html_repr += cell.assemble_html(document, child_blocks, None)
+            html_repr += "</tr>"
+        html_repr += "</tbody></table>"
+        return html_repr
```

Comment:
looks like somehow the headers are off here and other tables I've tested 
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/567e8a56-f07a-45ef-8b3e-6e6e8fe2f73a" />